app/models/custom_actions/actions/base.rb

```diff
@@ -98,6 +98,10 @@ def priority
 
   private
 
+  def deconstruct_keys(*)
+    { type:, custom_field: respond_to?(:custom_field) }
```

Comment:
:green_circle: What does the `custom_field` key indicate? `has_custom_field`? `is_custom_field`?

Whatever it is, I'd rename the key appropriately to show it. Right now I'd have expected `custom_field` to literally contain an element of type `CustomField`.

Reply:
Fair enough. I just mirrored the method name without giving further thought into it. =)